theano/gpuarray/dnn.py

```diff
@@ -3729,27 +3729,26 @@ def local_dnn_reduction(node):
     if node.inputs[0].ndim > 8:
         return
 
+    acc_dtype = node.op._acc_dtype(node.inputs[0].dtype)
```

Comment:
Ignoring the original acc_dtype is not good I think. If there isn,t one provided, it is good.

Reply:
We don't ignore the original.  `_acc_dtype` takes it into account : https://github.com/Theano/Theano/blob/42275cba95851ecca4d5580af9fe8bcbd2a3f481/theano/tensor/elemwise.py#L1747

Anyway, this is also what the non-cudnn code does so I don't see what's wrong.

Reply:
thanks for the explanation. Merging.